connectors/byoc.py

```diff
@@ -700,6 +700,10 @@ async def sync(self, sync_job_index, source_klass, elastic_server, bulk_options)
             await data_provider.ping()
             await asyncio.sleep(0)
 
+            logger.debug(f"Validating configuration for {data_provider}")
```

Comment:
What will the output of `data_provider` look like in that log message? Do we've access to the connector id and the service type maybe? Would probably make debugging easier later on as we can distinguish, which configuration was validated.

Reply:
Example log line:

```
[FMWK][11:50:15][DEBUG] Validating configuration for Datasource `MySQL`
```

As for the connector id and service type - I believe it's something that needs to be addressed in a logging epic. Log lines should work depending on context - if log happens inside connector, we need to output connector id/type, if inside job - also job id.